src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke.exceptions;
+
+/**
+ * Represents exceptions created from Duke bot operations.
+ */
+public class DukeException extends Exception {
+    public DukeException(String message) {
+        super(message);
+    }
+
+    public static DukeException missingInput(String taskName) {
```

Comment:
Should this method have a header comment since it is a public method? (in line with Java coding standards)